lib/api/controllers/documentController.js

```diff
@@ -67,15 +67,35 @@ class DocumentController extends NativeController {
    */
   async search (request) {
     const { from, size, scrollTTL, searchBody } = request.getSearchParams();
-    const { index, collection } = request.getIndexAndCollection();
+    const index = request.input.args.index;
+    const collection = request.input.args.collection;
```

Comment:
Because I do not want the search to throw if there is no index or collection specified because it could happend when there is only the target argument